demos/erc20/scripts/erc20-balance-user2.sh

```
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+source .local.env
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)